exercises/nodejs/src/categories.js

```diff
@@ -0,0 +1,4 @@
+export const POP_CATEGORY = 'Pop'
+export const SCIENCE_CATEGORY = 'Science'
+export const SPORT_CATEGORY = 'Sports'
+export const ROCK_CATEGORY = 'Rock'
```

Comment:
Also a great change, keeps the knowledge seperate and makes it highly more readable